ocp_resources/pod.py

```diff
@@ -167,3 +166,6 @@ def node(self):
     @property
     def ip(self):
         return self.instance.status.podIP
+
+    def _get_kube_v1_api(self):
```

Comment:
This should be moved to resource.py, that way it could be reused here as well as in _collect_pod_logs()